content/docs/tasks/security/authn-policy.md

```diff
@@ -249,6 +249,62 @@ $ kubectl exec $(kubectl get pod -l app=sleep -n legacy -o jsonpath={.items..met
 200
 ```
 
+## Enable mTLS incrementally
+
+Sometimes it's difficult to enable mTLS for a service without breaking any clients. For example, a service can have clients without Envoy sidecar. Those clients
+can only send plain text traffic.
+
+To address this, Authentication Policy mTLS has two modes:
+1. `STRICT` mode. A service in this mode only accepts TLS traffic. This is the default mode.
+1. `PERMISSIVE` mode. A service in this mode accepts both TLS and plain text request on the same port.
+
+Create the `PERMISSIVE` Authentication policy for httpbin,
+```bash
+cat <<EOF | istioctl create -n foo -f -
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "example-httpbin-permissive"
+spec:
+  targets:
+  - name: httpbin
+    namespace: foo
```

Comment:
namespace is under metadata, or you can skip it as it will use namespace from command line args

Reply:
Done.